src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,116 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+
+    @Override
+    public Driver create(Driver driver) {
+        String query = "INSERT INTO drivers (name, licence_number) "
+                + "VALUES (?, ?)";
```

Comment:
я так понимаю, что это не критично, но хороший тон в конце запроса передавать кавычки.
посмотри все запросы.
```suggestion
                + "VALUES (?, ?);";
```

Reply:
up to you. тут всего один запрос. несколько запросов подряд надо разделять, а тут нет критической необходимости